### PR TITLE
#161281811 Fix bugs on login

### PR DESCRIPTION
**Pull request by Estaer**
#### What does this PR do?
* Fixes bugs on Login

#### Description of Tasks to be completed?
*  Add token to local storage
*  Remove irrelevant password validations

#### What are the relevant pivotal tracker stories?

[#161281811](https://www.pivotaltracker.com/story/show/161281811)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
 import NavBar from './components/navigation/NavBar';
 import PageLoader from './components/PageLoader/PageLoader';
-import Login from './views/Login';
 
 const App = () => {
 	return (
 		<Router>
 			<div>
-				<NavBar/>
+				<NavBar />
```
Comment (isheebo) [inline src/App.js]: kindly revert this

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
-			<Switch>
-				<Route path='/' component={Home} exact/>
-				<Route path='/login' component={Login}/>
-			</Switch>
-				</div>
-		</Router>
-
-	);
+				<Switch>
+					<Route path='/' component={LandingPage} exact/>
```
Comment (isheebo) [inline src/App.js]: and this too

File: src/App.js (hunk 1 above)
Comment (Estaer) [reply src/App.js]: I reverted this.

File: src/App.js (hunk 2 above)
Comment (Estaer) [reply src/App.js]: I reverted this

**Comment by DrKimpatrick** [comment]
Add Heroku link to the PR. Am failing to log in even with correct credentials. 
image
That's the error displayed in the console.

**Pull request merged**